Documentation/gettingstarted/bgp-control-plane.rst

```diff
@@ -0,0 +1,330 @@
+      error and remove any existing BGP sessions. Only (n == 1) policies
+      **must** match a node's label sets.
+   -  Administrators should test a new BGP topology in a staging
+      environment before making permanent changes in production.
+
+-  Within a ``CiliumBGPPeeringPolicy`` each ``CiliumBGPVirtualRouter``
+   defined must have a unique ``localASN`` field.
+
+   -  A node cannot host two or more logical routers with the same local
+      ASN. Local ASNs are used as unique keys for a logical router.
+   -  A node can define the remote ASN on a per-neighbor basis to
+      mitigate this scenario. See ``CiliumBGPNeighbor`` CR
+      sub-structure.
+
+-  IPv6 single stack deployments **must** set an IPv4 encoded
+   ``routerID`` field in each defined ``CiliumBGPVirtualRouter`` object
+   within a ``CiliumBGPPeeringPolicy``
+
+   -  Cilium running on a IPv6 single stack cluster cannot reliably
+      generate a unique 32 bit BGP router ID, as it defines no unique
+      IPv4 addresses for the node. The administrator must define these
+      IDs manually or an error applying the policy will occur.
+   -  This is explained further in `Virtual Router
+      Attributes <#Virtual%20Router%20Attributes>`__
+
+Defining Topology
+~~~~~~~~~~~~~~~~~
+
+Within a ``CiliumBGPPeeringPolicy`` multiple
+``CiliumBGPVirtualRouter``\ (s) can be defined.
+
+Each one can be thought of as a logical BGP router instance.
+
+Defining more than one ``CiliumBGPVirtualRouter`` in a
+``CiliumBGPVirtualRouter`` creates more than one logical BGP router on
+the hosts which the policy matches.
+
+It is possible to create a single ``CiliumBGPPeeringPolicy`` for all
+nodes by giving each node in a cluster the same label and defining a
+single ``CiliumBGPPeeringPolicy`` which applies to this label.
+
+It is also possible to provide each ``Kubernetes`` node its own
+``CiliumBGPPeeringPolicy`` by giving each node a unique label and
+creating a ``CiliumBGPPeeringPolicy`` for each unique label.
+
+This allows for selecting subsets of nodes which peer to a particular
+BGP router while another subset of nodes peer to a separate BGP router,
+akin to an "AS-per-rack" topology.
+
+Virtual Router Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A ``CiliumBGPPeeringPolicy`` can apply to multiple nodes.
+
+When a ``CiliumBGPPeeringPolicy`` applies to one or more nodes each node
+will instantiate one or more BGP routers as defined by the list of
+``CiliumBGPVirutalRouter``.
+
+However, there are times where fine-grained control over an instantiated
+virtual router's configuration needs to take place.
+
+To accomplish this a Kubernetes annotation is defined which applies to
+Kubernetes Node resources.
+
+A single annotation is used to specify a set of configuration attributes
+to apply to a particular virtual router instantiated on a particular
+host.
+
+The syntax of the annotation is as follows:
+
+::
+
+       cilium.io/bgp-virtual-router.{asn}="key=value,..."
```

Comment:
Is it possible to set the `router-id` as a node annotation with this for single-stack ipv6? It might make it a bit less painful